src/main/webapp/jsp/instructorFeedbackResultsTop.jsp

```diff
@@ -51,10 +51,10 @@
           </div>
           <div class="col-sm-2">
               
-              <div class="form-group">
+              <div class="form-group"  style="padding-right:15px;">
```

Comment:
Maybe try to use bootstrap to align things instead? padding-right may cause alignment issues when the page is viewed on smaller screens.